examples/demo-simple-stf/src/lib.rs

```diff
@@ -26,7 +34,7 @@ impl<Vm: Zkvm, B: BlobReaderTrait> StateTransitionFunction<Vm, B> for CheckHashP
     type TxReceiptContents = ();
 
     // This is the type that will be returned as a result of `apply_blob`.
-    type BatchReceiptContents = ApplyBlobResult;
+    type BatchReceiptContents = ApplySlotResult;
```

Comment:
It feels like associated type needs to be renamed, but it is out of scope of this PR